5_databases/week3/Additional-queries.sql

```diff
@@ -0,0 +1,44 @@
+use homework3;
+--1 Get meals that has a price smaller than a specific price fx 90--
+SELECT *
+FROM `Meal`
+WHERE price < 90;
+
+--2 Get meals that still has available reservations--
+SELECT Meal.*
+FROM Meal JOIN Reservation  ON Meal.id = Reservation.meal_id
+WHERE  Reservation.created_date > NOW();
```

Comment:
This is a hard query to make. You need to take into account the max_reservations and the total sum of guests of the related reservations. Try to finish the query below. 
```sql
SELECT meal.*, SUM(???) AS total_reservations
FROM meal
LEFT JOIN reservation ON meal.id = reservation.meal_id
GROUP BY meal.id
HAVING meal.max_reservations > ???;
```